### PR TITLE
Add notes about multiple dates and examples for update method

### LINKED ISSUE #2038: Update Multiple Dates programmatically not working
### Expected behaviour

I am storing expected selected dates in a variable. Then passing this variable to update dates, its not selecting dates.
### Datepicker version used

ex. 1.6.1.
### Example code

var dta = '"10/19/2016","10/20/2016"';
$('#datepicker').datepicker('update',dta);

### PR DESCRIPTION
**Pull request by Azaret**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | fixes #2038

### CODE REVIEW

**Comment by acrobat** [comment]
Just one little comment, extra info looks good!

File: docs/methods.rst [hunk 1]
```diff
@@ -39,17 +39,21 @@ update
...
+* date (String|Date|Array, optional)
 * date (String|Date, optional)
+* ...
 
-Update the datepicker with given argument or the current input value.
-
-If ``date`` is provided and is a Date object, it is assumed to be a "local" date object, and will be converted to UTC for internal use.
+Update the datepicker with given arguments or the current input value.
+The arguments can be either an array of strings, an array of Date objects, multiples strings or multiples Date objects.
+If ``date`` arguments are provided and are Date objects, it is assumed to be "local" Date objects, and will be converted to UTC for internal use.
```
Comment (Azaret) [inline docs/methods.rst]: Kk

**Comment by acrobat** [comment]
Thanks @Azaret

**Comment by Azaret** [comment]
Thank to you @acrobat 😉

File: docs/methods.rst [hunk 2]
```diff
@@ -39,17 +39,21 @@ update
 
 Arguments:
 
+* date (String|Date|Array, optional)
 * date (String|Date, optional)
```
Comment (vsn4ik) [inline docs/methods.rst]: @Azaret I think that you want to delete this line and leave `* date (String|Date|Array, optional)`.

Comment (Azaret) [inline docs/methods.rst]: Well I wasn't sure how to show the fact that date is `*args` for `string` and `Date`. But not when `Array`.

**Pull request merged**